ci-docker-images/oci-volume-provisioner-system-test/Makefile

```diff
@@ -12,17 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER_REGISTRY ?= iad.ocir.io
+OCIRREGISTRY ?= iad.ocir.io
+OCIRTENNANCY ?= oracle
 DOCKER_IMAGE_NAME ?= oci-volume-provisioner-system-test
 DOCKER_IMAGE_TAG ?= 1.0.2
-
-.PHONY: build
+.PHONY: build 
 build:
-	docker build --build-arg http_proxy=${http_proxy} \
-		         --build-arg https_proxy=${https_proxy} \
-				 -t ${DOCKER_REGISTRY}/${DOCKER_REGISTRY_USERNAME}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} .
+	docker build -t $(OCIRREGISTRY)/$(OIRTENNANCY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) .
 
 .PHONY: push
 push: build
-	docker login -u '$(DOCKER_REGISTRY_USERNAME)' -p '$(DOCKER_REGISTRY_PASSWORD)' $(DOCKER_REGISTRY)
-	docker push ${DOCKER_REGISTRY}/${DOCKER_REGISTRY_USERNAME}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}
+	@docker login -u '$(OCIRRUSERNAME)' -p '$(OCIRRPASSWORD)' $(OCIRREGISTRY) 2> /dev/null
```

Comment:
If we wanted to be really picky, I recon it'll be clearer if we do:
@docker login -u '$(OIRTENNANCY)/$(OCIRRUSERNAME)' -p '$(OCIRRPASSWORD)' $(OCIRREGISTRY) 2> /dev/null
i.e. the username then really is the oci username, and not a combination of tenancy/username.